quiche/src/tls.rs

```diff
@@ -1093,6 +1095,24 @@ fn map_result_ptr<'a, T>(bssl_result: *const T) -> Result<&'a T> {
     }
 }
 
+fn set_local_error_if_needed(
```

Comment:
Hi!

Why is this function defined outside of `impl Handshake`?

If it was an associated function (defined inside impl ...) you could then just call like:<br>
`self.set_local_error_if_needed(ex_data, rc);`

Reply:
Yeah, better make this a method of `Handshake` (`map_result_ssl()` should probably be too, but that's a separate issue).

Also, maybe call this `set_transport_error()` or something.